portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,93 @@
+let answer = 0;
+
+function loadInTruth() {
+
+    let firstTruth = Math.floor(Math.random() * truths.length);
+    let secondTruth = Math.floor(Math.random() * truths.length);
+    while (firstTruth == secondTruth && truths.length != 1) {
```

Comment:
The second half of this while condition isn't necessary, is it?

Reply:
Removed second condition